file.js

```diff
@@ -14,3 +17,84 @@ document.querySelectorAll('.link').forEach((n) => n.addEventListener('click', ()
   navbar.classList.remove('active');
   container.classList.remove('active');
 }));
+// popup window
+const myProjects = [
+  {
```

Comment:
- [ ] Great job getting your popups from the array. Please, kindly make your background photo bigger, so it looks like the figma version.
- [ ] please remove the extra with from your `see live` and `see source` buttons and ensure they follow the figma design
your design
![Screenshot (260)_LI](https://user-images.githubusercontent.com/65298048/180022441-ae90223d-575b-431e-baef-6c9910602f70.jpg)
figma design
![Screenshot (259)_LI](https://user-images.githubusercontent.com/65298048/180022869-e4d522e4-eeda-439b-80ac-68e8228f96e6.jpg)